From ab25c18f07a252b827a9b34eb71d589be9c06ec9 Mon Sep 17 00:00:00 2001
From: Janmm14 <computerjanimaus@yahoo.de>
Date: Sun, 17 Jan 2016 17:22:05 +0100
Subject: Add origin and patcher git hash instead of patched git hash to
 version by using env variables which are set in build.sh

Remove scriptus maven plugin, as its no longer needed.
---
 pom.xml | 19 +++----------------
 1 file changed, 3 insertions(+), 16 deletions(-)

diff --git a/pom.xml b/pom.xml
index ec0ace3..e7c00d6 100644
--- a/pom.xml
+++ b/pom.xml
@@ -75,6 +75,9 @@
         <maven.compiler.source>1.7</maven.compiler.source>
         <maven.compiler.target>1.7</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <origin_commit>unknown</origin_commit>
+        <patcher_commit>unknown</patcher_commit>
+        <describe>git:FlexPipe:${project.version}:origin-${origin_commit}-patcher-${patcher_commit}:${build.number}</describe>
     </properties>
 
     <dependencies>
@@ -107,22 +110,6 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>net.md-5</groupId>
-                <artifactId>scriptus</artifactId>
-                <version>0.3.1</version>
-                <configuration>
-                    <format>git:${project.name}:${project.version}:%s:${build.number}</format>
-                </configuration>
-                <executions>
-                    <execution>
-                        <phase>initialize</phase>
-                        <goals>
-                            <goal>describe</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>animal-sniffer-maven-plugin</artifactId>
                 <version>1.13</version>
-- 
1.9.5.msysgit.0

